### ISSUE TITLE
No help for turtle in Python & JS when right-click

### ISSUE THREAD

**Issue by Sheila111** [microsoft/pxt-microbit#3140]
**Describe the bug**
No help for **turtle** in Python & JS when right-click

**Steps to reproduce the behavior**
1.Navigate to https://makecode.microbit.org/beta#editor
2.Turtle---->Square--->Start Tutorial
3.Right click----->help

**Expect behavior**
Show help for **turtle** in Python & JS

**Actual behavior**
No help for **turtle** in Python & JS when right-click
![1](https://user-images.githubusercontent.com/59192069/82642942-499c2a80-9c41-11ea-8bd2-4f2882bace25.gif)

**Additional context**
1.OS: Windows(rs6)
2.makecode.microbit.org version: 2.3.68
3.Microsoft MakeCode version: 5.37.82

**Comment by pelikhan** [comment]
Help should go to ``https://makecode.microbit.org/pkg/microsoft/microbit-turtle#namespace-function`` like the blocks.

**Comment by abchatra** [comment]
@ganicke I believe @riknoll fixed most of the issues. If you need any product fixes let me know.

**Comment by ganicke** [comment]
Help now goes to the heading content for the extension readme, like: https://makecode.microbit.org/pkg/Microsoft/pxt-microturtle#turtle-back.

**Comment by ganicke** [comment]
Nope, context help on the extension functions still doesn't work.

![image](https://user-images.githubusercontent.com/27789908/145899083-7dde9f35-7c5f-4ae7-bfc6-69d3de0ab6c2.png)

**Comment by abchatra** [comment]
I can reproduce this issue.
